src/Handshake/ServerNegotiator.php

```diff
@@ -81,7 +99,7 @@ public function handshake(RequestInterface $request) {
             }, null);
 
             if ($this->_strictSubProtocols && null === $match) {
-                return new Response(400, [], '1.1', null ,'No Sec-WebSocket-Protocols requested supported');
```

Comment:
Can you keep the previous "No Sec-WebSocket-Protocols requested supported" reason in the response please?